diff -urbB kernel.unpatched/debian.master/config/annotations kernel.aaeon-iwmi-wdt/debian.master/config/annotations
--- kernel.unpatched/debian.master/config/annotations	2022-08-04 17:16:56.000000000 +0000
+++ kernel.aaeon-iwmi-wdt/debian.master/config/annotations	2022-08-13 01:21:29.477633005 +0000
@@ -10485,7 +10485,7 @@
 CONFIG_SBC_EPX_C3_WATCHDOG
 CONFIG_INTEL_MEI_WDT                            policy<{'amd64': 'm'}>
 CONFIG_NI903X_WDT                               policy<{'amd64': 'm'}>
 CONFIG_NIC7018_WDT                              policy<{'amd64': 'm'}>
-CONFIG_AAEON_IWMI_WDT                           policy<{'amd64': 'm'}>
+CONFIG_AAEON_IWMI_WDT                           policy<{'amd64': '-'}>
 CONFIG_BCM2835_WDT                              policy<{'arm64': 'm'}>
 CONFIG_BCM7038_WDT                              policy<{'arm64': 'm'}>
 CONFIG_MEN_A21_WDT                              policy<{'amd64': 'm', 'arm64': 'm', 'armhf': 'm', 'ppc64el': 'm', 's390x': 'm'}>
