diff -urbB kernel.unpatched/debian.master/config/annotations kernel.dwarf/debian.master/config/annotations
--- kernel.unpatched/debian.master/config/annotations	2022-08-12 12:20:47.401479000 +0000
+++ kernel.dwarf/debian.master/config/annotations	2022-08-04 17:16:56.000000000 +0000
@@ -11730,8 +11730,8 @@
 CONFIG_DEBUG_INFO_SPLIT
 CONFIG_DEBUG_INFO_BTF                           flag<REVIEW> note<Needs newer pahole for armhf>

 # Menu: Kernel hacking >> Compile-time checks and compiler options >> Compile the kernel with debug info >> DWARF version
-CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT       policy<{'amd64': 'n', 'arm64': 'n', 'armhf': 'n', 'ppc64el': 'n', 's390x': 'n'}>
-CONFIG_DEBUG_INFO_DWARF5                        policy<{'amd64': 'y', 'arm64': 'y', 'armhf': 'y', 'ppc64el': 'y', 's390x': 'y'}>
+CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT       policy<{'amd64': 'y', 'arm64': 'y', 'armhf': 'y', 'ppc64el': 'y', 's390x': 'y'}>
+CONFIG_DEBUG_INFO_DWARF5                        policy<{'amd64': 'n', 'arm64': 'n', 'armhf': 'n', 'ppc64el': 'n', 's390x': 'n'}>

 # Menu: Kernel hacking >> Debug Oops, Lockups and Hangs
 CONFIG_PANIC_ON_OOPS                            policy<{'amd64': 'n', 'arm64': 'n', 'armhf': 'n', 'ppc64el': 'n', 's390x': 'n'}>
diff -urbB kernel.unpatched/debian.master/config/config.common.ubuntu kernel.dwarf/debian.master/config/config.common.ubuntu
--- kernel.unpatched/debian.master/config/config.common.ubuntu	2022-08-12 12:20:47.401479000 +0000
+++ kernel.dwarf/debian.master/config/config.common.ubuntu	2022-08-04 17:16:56.000000000 +0000
@@ -2592,7 +2592,7 @@
 CONFIG_DEBUG_INFO=y
 CONFIG_DEBUG_INFO_BTF_MODULES=y
 # CONFIG_DEBUG_INFO_COMPRESSED is not set
 # CONFIG_DEBUG_INFO_DWARF4 is not set
-CONFIG_DEBUG_INFO_DWARF5=y
+CONFIG_DEBUG_INFO_DWARF5=n
 # CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT is not set
 # CONFIG_DEBUG_INFO_REDUCED is not set
 # CONFIG_DEBUG_INFO_SPLIT is not set
