diff -urbB kernel.unpatched/debian.master/config/annotations kernel.sensors-aaeon/debian.master/config/annotations
--- kernel.unpatched/debian.master/config/annotations	2023-07-05 18:56:36.151256013 +0000
+++ kernel.aaeon-iwmi-wdt/debian.master/config/annotations	2023-07-05 19:03:27.153523008 +0000
@@ -10651,7 +10651,7 @@
 CONFIG_SELECT_MEMORY_MODEL                      policy<{'amd64': 'y', 'armhf': 'y', 'ppc64el': 'y'}>
 CONFIG_SENSIRION_SGP30                          policy<{'amd64': 'm', 'arm64': 'm', 'armhf': 'm', 'ppc64el': 'm'}>
 CONFIG_SENSIRION_SGP40                          policy<{'amd64': 'm', 'arm64': 'm', 'armhf': 'm', 'ppc64el': 'm'}>
-CONFIG_SENSORS_AAEON                            policy<{'amd64': 'm'}>
+CONFIG_SENSORS_AAEON                            policy<{'amd64': '-'}>
 CONFIG_SENSORS_ABITUGURU                        policy<{'amd64': 'm'}>
 CONFIG_SENSORS_ABITUGURU3                       policy<{'amd64': 'm'}>
 CONFIG_SENSORS_ACPI_POWER                       policy<{'amd64': 'm', 'arm64': 'm'}>
